server.js

```diff
@@ -0,0 +1,5 @@
+const app = express();
```

Comment:
This doesn't work-- you have to import the app here, not instantiate a new Express server.  You want 
 ```suggestion
const app = require('./lib/app')
```
Now, when you run `npm start` your server will start to run.